back/app/controllers/omniauth_callback_controller.rb

```diff
@@ -173,6 +173,7 @@ def update_user!(auth, user, authver_method)
 
     attrs = authver_method.updateable_user_attrs
     update_hash = authver_method.profile_to_user_attrs(auth).slice(*attrs).compact
+    user.confirm # confirm user email if not already confirmed
```

Comment:
If a user has been created through other methods but has not confirmed, then we need to confirm here